src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -233,6 +233,48 @@ public void CounterPayloadReceived(CounterPayload payload, bool pauseCmdSet)
             }
         }
 
+        public void CounterStopped(CounterPayload payload)
+        {
+            lock (_lock)
+            {
+                string providerName = payload.ProviderName;
+                string counterName = payload.Name;
+                string tags = payload.Tags;
+
+                if (!providers.TryGetValue(providerName, out ObservedProvider provider))
```

Comment:
Why doesn't this class use our normal naming scheme? Shouldn't `providers` be `_providers` since it is a field on the class? Looking at this file, it looks like we randomly do or don't follow convention

Reply:
I doubt there is an intentional reason it doesn't follow the naming scheme, someone likely just overlooked it in the past.